hydroflow/src/scheduled/mod.rs

```diff
@@ -51,6 +52,8 @@ pub struct Hydroflow {
     ready_queue: VecDeque<SubgraphId>,
     event_queue_send: SyncSender<SubgraphId>, // TODO(mingwei) remove this, to prevent hanging.
     event_queue_recv: Receiver<SubgraphId>,
+
+    pub rt: Rc<tokio::runtime::Runtime>,
```

Comment:
This was mostly here so that we could make use of tokio's nicer wrappers around networking codecs. I don't really have a clear picture of how Hydroflow fits into the tokio world so if you have strong opinions on that would like to hear them.